common/djangoapps/terrain/stubs/tests/test_video.py

```diff
@@ -0,0 +1,44 @@
+"""
+Unit tests for Video stub server implementation.
+"""
+import unittest
+import requests
+from ..video_source import VideoSourceHttpService
```

Comment:
I would import this as `from terrain.stubs.video_source import VideoSourceHttpService`